scripts/populate_jira_cache.py

```diff
@@ -0,0 +1,82 @@
+# /// script
+# requires-python = ">=3.13"
+# dependencies = [
+#     "click",
+# ]
+# ///
+from pathlib import Path
+import re
+
+import click
+
+from robottelo.utils.issue_handlers.jira import get_data_jira, jira_cache
```

Comment:
How will `uv` import these dependencies if we run it in an isolated environment using `uv run`?

Reply:
unfortunately, I don't think this is something that plays nicely with `uv run`. It may be possible to do some path manipulation at the start of the script to allow it to be executed with `uv run`, but I considered that to be too janky at this time.

For now, I expect us to just execute it like a normal python script. Good catch